bin/pre-commit.sh

```diff
@@ -1,37 +1,23 @@
 #!/bin/bash
 
-if [ -n "$failIt" ]; then
-  echo "Expecting failIt to be empty."
-  exit 1
-fi
+rc=0
```

Comment:
Is there a reason why we use `rc` to track return codes, rather than using `set -e` to bail out early on problems?

Reply:
Yeah, as monopole mentionned earlier in this PR, we want to run all the tests, even if one fails. So that you don't have to fix-repush-look for failures repeatedly ...